src/libs/actions/IOU.js

```diff
@@ -13,7 +16,117 @@ function getPreferredCurrency() {
     }, 1600);
 }
 
-// Re-enable the prefer-default-export lint when additional functions are added
+/**
+ * Creates IOUSplit Transaction
+ * @param {Object} parameters
+ * @param {String} parameters.amount
+ * @param {String} parameters.comment
+ * @param {String} parameters.currency
+ * @param {String} parameters.debtorEmail
+ */
+function createIOUTransaction({
+    comment, amount, currency, debtorEmail,
+}) {
+    Onyx.merge(ONYXKEYS.IOU, {loading: true, creatingIOUTransaction: true, error: false});
+    let iouReportID = '';
+    API.CreateIOUTransaction({
+        comment,
+        amount,
+        currency,
+        debtorEmail,
+    })
+        .then((data) => {
+            iouReportID = data.reportID;
+            return iouReportID;
```

Comment:
Just return data.reportID?

Reply:
@Julesssss  we can definitely return just data.reportID here, but we'd still need to set iouReportID since we use it in a component down in the order.

Reply:
Oh yes of course 😄